api_app/analyzers_manager/observable_analyzers/greynoise_labs.py

```diff
@@ -104,7 +112,9 @@ def _update_db(cls, auth_token: str):
         }
 
         try:
-            logger.info("Fetching data from greynoise API (Greynoise_Labs).....")
+            logger.info(
+                f"Fetching data from greynoise API ({cls._get_observable_name()})....."
```

Comment:
it's enough to use `self.observable_name` cause it is inherited from the base class :P

Reply:
but _update_db is class method and it cannot access observabl_name. I have already trie it but does not work. @mlodic

Reply:
ah you are right, my bad. there is no need of anything else in the log so, you can revert to the previous message

Reply:
hardcoded? @mlodic